templates/download/index.html

```diff
@@ -4,55 +4,87 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<div class="p-strip is-shallow">
-  <div class="row">
-    <div class="col-12">
-      <h1>Ubuntu downloads</h1>
+<section class="p-strip--light u-no-padding--top">
+  <div class="p-strip-suru-half-top">
```

Comment:
You shouldn't double strip here. This suru strip is extending p-strip so you should be able to just replace `p-strip--light` above

Reply:
I cannot figure out how to make that work without messing up the design and there is another suru strip that does the same thing.

Sorry.